histogram/src/config.rs

```diff
@@ -130,6 +130,11 @@ impl Config {
         self.max_value_power
     }
 
+    /// Returns the relative error (in percentage) of this configuration.
+    pub fn error(&self) -> f64 {
+        100.0 / 2_u64.pow(self.grouping_power as u32) as f64
```

Comment:
Is this formula correct when `grouping_power == max_value_power - 1`? I believe in this case the error should be zero.

Reply:
Yes, it only applies to the logarithmic bins. Fixed the code and comment to reflect that linear bins have zero error.